README.md

```diff
@@ -10,22 +10,33 @@ with this repository have an IREE build setup in an `iree-build` directory that
 is also a sibling. This is not a requirement, but instructions will need to be
 changed for different paths.
 
-Preparing repostiory:
+Preparing repository:
 
 ```
 git submodule update --init
 ```
 
 ## Enabling in IREE
 
+Currently (12/12/2023) we are waiting for an MLIR fix in the transform dialect,
+and so cannot build with the latest IREE (and its LLVM submodule). To pin IREE
+and its submodules to a amd-aie compatible version, run
```

Comment:
maybe above three lines should be an issue since we want end users to run sync_deps.py regardless of the current transform dialect issue, so that the project builds for them consistently.

Reply:
Ok, I thought this was just a temporary fix. I will update to say that running sync_deps.py is the expected workflow for this plugin